cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -610,16 +611,10 @@ private class ServiceImpl(
         ccpaApplies: Boolean?,
         usNatApplies: Boolean?,
     ): Either<ConsentStatusResp> {
-        val csParams = messageReq.toConsentStatusParamReq(
-            gdprUuid = campaignManager.gdprUuid,
-            ccpaUuid = campaignManager.ccpaUuid,
-            usNatUuid = campaignManager.usNatConsentData?.uuid,
-            localState = campaignManager.messagesOptimizedLocalState,
-        )
+        val csParams = messageReq.toConsentStatusParamReq(campaignManager)
 
         return getConsentStatus(csParams)
             .executeOnRight {
-                dataStorage.updateLocalDataVersion()
```

Comment:
Just realised all code regarding local data version was removed. Is that on purpose?